coopr-docs/docs/source/guide/superadmin/docker-automator-plugin.rst

```diff
@@ -0,0 +1,119 @@
+..
+   Copyright © 2012-2015 Cask Data, Inc.
```

Comment:
It can. It can also be considered a derivative work of the `shell-automator-plugin.rst`, which would make both correct. I generally use a new starting date on files "created from scratch" but keep the original date when doing `s/oldvalue/newvalue/g` style creation of new files. If it matters to you, I can change it.